### DISCUSSION TITLE
Where can I find settings to stop upload to peers who don't allow downloads?

### DISCUSSION THREAD

**Discussion by itsmnthn** [transmission/transmission#6996] [General]
I have connected to peers who is downloading only no upload at all. I'm stuck at 25% download for a quite long and people just don't allow me to download and all the speed goes to upload. I'm okay with uploading unlimited as long as I can download unlimited.

**Comment by andreygursky** [comment]
> I'm stuck at 25% download for a quite long ...

It looks like a peer with 100% is absent for that time and those peers just do not have any new/different pieces of data than you, that's why there is nothing to upload to you. But if you notice those peers have downloaded more than 25%, then there is indeed some issue.
